include/LightGBM/config.h

```diff
@@ -268,6 +268,12 @@ struct Config {
   // desc = random seed for bagging
   int bagging_seed = 3;
 
+  // alias = sub_feature_bynode, colsample_bytree_bynode
+  // desc = set to ``true`` will randomly select part of features for each node
```

Comment:
It seems to me that `set to` -> `set this to` and `will` -> `to` would be better wording.